lib/vscode/src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -406,10 +406,44 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 		this.root = parent;
 
 		const overlay = append(this.root, $('.overlay'));
+
```

Comment:
was this intended?

Reply:
I think I was just adding space to make it more readable, but I'll clean it up